src/Aspire.Dashboard/Components/ResourcesGridColumns/LogLevelColumnDisplay.razor.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using Aspire.Dashboard.Otlp.Model;
+using Aspire.Dashboard.Resources;
+using Microsoft.JSInterop;
+
+namespace Aspire.Dashboard.Components;
+
+public partial class LogLevelColumnDisplay
+{
+    private bool TryGetErrorInformation([MaybeNullWhen(false)] out string errorInfo)
```

Comment:
```suggestion
    private bool TryGetErrorInformation([NotNullWhen(true)] out string? errorInfo)
```
Isn't this the idomatic way to do this? I've never seen the opposite.